### PR TITLE
Removing new from refactored methods

### PR DESCRIPTION
**Pull request by Andrea-Campanella**

### CODE REVIEW

File: pkg/manager/setconfig.go [hunk 1]
```diff
@@ -99,29 +99,29 @@ func (m *Manager) computeNewNetworkConfig(targetUpdates map[string]devicechanget
 		//FIXME this is a sequential job, not parallelized
 		version := deviceInfo[devicetype.ID(target)].Version
 		deviceType := deviceInfo[devicetype.ID(target)].Type
-		newChange, err := m.ComputeNewDeviceChange(
```
Comment (SeanCondon) [inline pkg/manager/setconfig.go]: I thought `newChange` here was actually a good name for the **new** change

File: pkg/cli/networkchanges_test.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func generateNetworkChangeData(count int) {
 	now := time.Now()
 
 	for cfgIdx := range networkChanges {
-		networkID := fmt.Sprintf("a_new_network_change-%d", cfgIdx)
```
Comment (SeanCondon) [inline pkg/cli/networkchanges_test.go]: I thought the existing name was fine for a **new** network change

File: pkg/manager/setconfig.go (hunk 1 above)
Comment (Andrea-Campanella) [reply pkg/manager/setconfig.go]: brought back

File: pkg/cli/networkchanges_test.go (hunk 2 above)
Comment (Andrea-Campanella) [reply pkg/cli/networkchanges_test.go]: reverted

**Pull request merged**